flows/editor.md

```diff
@@ -65,6 +65,24 @@ If we modify the If node's config so the statement is `search.Count < 1` and re-
 
 ![](debug-if-false.png)
 
+## Enabled vs Disabled Nodes
+
+Individual nodes in a flow can be disabled in order to execute parts of a flow without running the entire flow. This feature can be useful for troubleshooting or debugging a flow. For example, one may disable an Email node while troubleshooting a flow in order to avoid sending unnecessary emails.
+
+Nodes are enabled by default. Any node can be disabled or re-enabled through its right-click menu or the configuration pane:
+
+![Nodes can be disabled via the right-click menu or configuration pane](disable-node.png)
+
+When a node is disabled, it blocks its own execution and the execution of all downstream nodes. 
+
+Consider the following flow. The PDF node is grayed out and has a ⛔ badge which indicates that it has been disabled. The Email and Slack File nodes which are downstream from the PDF node are also grayed out. As explained in the Design Issues pane (and in each node's tooltip), those nodes are disabled because each has a disabled node upstream.
```

Comment:
```suggestion
Consider the following flow: The PDF node is grayed out and has a ⛔ badge which indicates that it has been disabled. The Email and Slack File nodes which are downstream from the PDF node are also grayed out. As explained in the Design Issues pane (and in each node's tooltip), those nodes are disabled because each has a disabled node upstream.
```